src/commands/resumes/getResume.ts

```diff
@@ -0,0 +1,104 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+
+import { default as resumes } from './resumes.json';
+
+import { availableDomains } from '../../components/interview';
+
+const RESUME_EMBED_COLOUR = '#77bb00';
+
+interface Resume {
+  url: string;
+  author: string;
+}
+
+/*
+ * Send embed for getting an example resume
+ */
+const getResumeEmbed = (
```

Comment:
actually, embed colours and helper functionss might live better in a resume/utils bc in the future there's def gonna be a lot of "helper" funcs